src/sdk/pynni/nni/platform/local.py

```diff
@@ -36,7 +36,7 @@
     os.makedirs(_outputdir)
 
 _nni_platform = os.environ['NNI_PLATFORM']
-if _nni_platform != 'pai':
+if _nni_platform not in ['pai', 'kubeflow']:
```

Comment:
add frameworkcontroller.

Reply:
Sure. Thanks for reminding. 
Already created an issue to track: https://github.com/Microsoft/nni/issues/552